cli/flox/src/config/mod.rs

```diff
@@ -35,8 +33,9 @@ pub struct Config {
     #[serde(default)]
     pub nix: Option<NixConfig>,
 
+    // You should prefer to use `Flox` over `Config` where available.
```

Comment:
doc: you can use `///` to create doc comments, i think it would be useful to explain the origin of `Features` and how to access / use them and have it show up in generated docs